packages/lesswrong/components/vulcan-core/App.tsx

```diff
@@ -40,18 +40,18 @@ const App = ({serverRequestStatus, timeOverride, history}: ExternalProps & {
   const locale = localeSetting.get();
 
   useEffect(() => {
-    void userChangedCallback.runCallbacks({
-      iterator: currentUser,
-      properties: [],
-    });
+    if (!bundleIsServer) {
+      const { onUserChanged } = require('@/client/logging');
+      onUserChanged(currentUser);
+    }
     moment.locale(locale);
   }, [currentUser, locale]);
 
   useEffect(() => {
-    void userChangedCallback.runCallbacks({
-      iterator: currentUser,
-      properties: [],
-    });
+    if (!bundleIsServer) {
+      const { onUserChanged } = require('@/client/logging');
+      onUserChanged(currentUser);
+    }
```

Comment:
Doesn't make sense for this PR but I think these two useEffects could be combined, and currently they result in `onUserChanged` being called twice when the client is first initialized